MANIFEST.in

```diff
@@ -11,15 +11,9 @@ recursive-include licenses *
 recursive-include cextern *
 recursive-include scripts *
 
-include pypeit/requirements.txt
```

Comment:
We should also delete/deprecate the requirements_doc.txt file at the top-level.

Reply:
yes, good point. i need to refactor readthedocs.yml as well to match that. was on my list and missed it. it's mostly boilerplate so it's pretty easy.

Reply:
requirements_doc.txt is now deprecated and readthedocs.yml updated to install via pip and include the `docs` extra requirements.